src/lib/storage/fixed_size_attribute_vector.hpp

```diff
@@ -0,0 +1,37 @@
+#pragma once
+
+// the linter wants this to be above everything else
+#include <types.hpp>
+#include <vector>
+#include "base_attribute_vector.hpp"
+
+namespace opossum {
+
+// Attribute vector with a fixed size that stores references to values of the given template type
```

Comment:
The term "reference to x" has a specific meaning in C++ and might be confusing here

Reply:
a58545cc38f8547b9af338326972908c28adc3a0